pkg/containerrun/containerrun.go

```diff
@@ -504,28 +582,58 @@ type Stdio struct {
 // ProcessRegistry handles all the processes.
 type ProcessRegistry struct {
 	processes []Process
+	timer     *time.Timer
 	sync.Mutex
 }
 
 // NewProcessRegistry constructs a new ProcessRegistry.
 func NewProcessRegistry() *ProcessRegistry {
 	return &ProcessRegistry{
 		processes: make([]Process, 0),
+		// It should always be safe to call timer.Stop(), so it must not be nil.
+		timer: time.NewTimer(time.Millisecond),
 	}
 }
 
+// Count returns the number of processes in the registry
+func (pr *ProcessRegistry) Count() int {
+	return len(pr.processes)
```

Comment:
Does this need to take a lock? Not that we care about the number that much, but mostly to ensure the race detector never makes a peep?